src/components/staking/PageValidators.vue

```diff
@@ -1,9 +1,6 @@
 <template>
   <TmPage
     :managed="true"
-    :loading="$apollo.queries.validators.loading"
-    :loaded="!$apollo.queries.validators.loading"
```

Comment:
if this property is set to true, the loader doesn't show anymore. I would recommend on first update to set a `loaded` property to true instead of removing the whole logic.